src/ToDoStatus.js

```diff
@@ -0,0 +1,66 @@
+import { editTodo, removeTodo } from './functionalities.js';
+import { myArray } from './index';// eslint-disable-line
+
+const todoMainContainer = document.querySelector('.todos-container');
+
+// get from localstorage function
+const getFromLocal = () => {
+  const data = JSON.parse(localStorage.getItem('list'));
+  data.map(i => {// eslint-disable-line
+    myArray.push(i);
+    const todoContainer = document.createElement('div');
+    todoContainer.className = 'todoContainer';
+    todoContainer.innerHTML += `
+      <input type='checkbox' class='checkbox'>
+      <span>${i.description}</span>
+      <i class='fas fa-ellipsis-v'></i>
+      <i class='fas fa-trash-alt'></i>
+    `;
+    todoMainContainer.appendChild(todoContainer);
+
+    const EditIcons = document.querySelectorAll('.fa-ellipsis-v');
+    EditIcons.forEach((i) => {
+      i.addEventListener('click', () => {
+        editTodo(todoContainer, i.previousElementSibling);
+        i.parentElement.classList.add('checkedContainer');
+      });
+    });
+  });
+
+  const checkbox = document.querySelectorAll('.checkbox');
+  checkbox.forEach((i) => {
```

Comment:
- [ ] Great job implementing all the task. Please, when a task is edited it should go back to normal rather than remaining highlighted and italic. Kindly fix that.
![Screenshot (252)_LI](https://user-images.githubusercontent.com/65298048/179116420-b1b8261a-3214-49f2-84ba-d1de9061874e.jpg)
- [ ] also, please make it possible for a task to be deleted even if it is not yet completed. When you double-click on the `:`, the task disappears, but the task space remains. This will enable someone to keep their to-do clean in case they change their mind about a particular task.